xpathwebdriver/browser.py

```diff
@@ -37,8 +40,8 @@ def _free_webdriver(self):
             self._browser_context = None
 
     def _quit_failed_webdriver(self):
+        # todo this code is trying to access  to a protected method
```

Comment:
Could we convert .is_failed_webdriver to a public method or provide some getters for that feature?